### ISSUE TITLE
Technical Issue #2: Dropdown menu / Navbar suggestions

### ISSUE THREAD

**Issue by nicoleedwardsp** [Hmulholland1/MyWebTechProject#5]
Instead of having the different steps of the cheesemaking process hidden under "pages", I'd suggest either naming this navbar item "how our cheese is made" or deleting the dropdown menu completely and having the different steps appear on the navbar individually. 

Also, I'd recommend having the contact and sitemap towards the end of the navbar.... since they are the "extra" pages. How the cheese is made and your family story should be the focus points of your website.

https://github.com/Hmulholland1/MyWebTechProject/blob/d0e4c23be44cee1f2c7bfabd8f962850cffb7f6a/index.html#L60

**Comment by Hmulholland1** [comment]
I agree with the dropdowns name but Im going to keep the rest of the navbar how it is
